tools/docker/README.md

```diff
@@ -208,3 +208,12 @@ echo "ALLOW_ORIGINS=http://localhost:1337" > chainlink-variables.env
 ```
 docker system prune
 ```
+
+## The build process takes up a lot of resources / brings my computer to a halt
+
+The default configuration tries to build everything in parallel.  You can get this by clearing our scripts docker compose's build options.
+
+```
+# Use docker compose's default build configuration
+export DOCKER_COMPOSE_BUILD_OPTS=""
```

Comment:
How do you feel about adding an example option here? e.g.

```
export DOCKER_COMPOSE_BUILD_OPTS="--parallel"
```

Reply:
Ohh I changed the goal of this PR to allow changing the build options.  So it still parallel builds by default.  Other systems rely on this behavior and it may be a rabbit hole.

Reply:
Ah, gotcha. So I need to set this env var on my machine.